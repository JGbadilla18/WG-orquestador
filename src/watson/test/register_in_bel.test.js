const testWatson = require('./testWatson');

watsonText = 'como ingreso banco en linea.'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
