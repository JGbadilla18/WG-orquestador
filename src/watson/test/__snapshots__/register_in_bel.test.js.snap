// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`como ingreso banco en linea. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "buttonType": "web_url",
      "buttons": Array [
        Object {
          "title": "Aquí",
          "url": "https://www.corporacionbi.com/gt/bancoindustrial/bi-en-linea",
        },
      ],
      "response_index": 0,
      "response_type": "button",
      "text": "Para ingresar a tu Bi en Línea sigue los siguientes tips de seguridad: Asegúrate que la página utiliza https, además de que aparezca el símbolo del candado relacionado con el certificado digital.
Conoce cómo ingresar a Bi en Línea",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
